server/client.go

```diff
@@ -132,7 +179,6 @@ func (c *client) initClient() {
 	c.cid = atomic.AddUint64(&s.gcid, 1)
 	c.bw = bufio.NewWriterSize(c.nc, startBufSize)
 	c.subs = make(map[string]*subscription)
-	c.debug = (atomic.LoadInt32(&debug) != 0)
```

Comment:
The `log` has 2 booleans `debug` and `trace`. The use of the booleans in the client structure was to speed up things in that we don't even call the logging if trace is not set. Logging itself would check log package trace, but it means that we have already passed all args to logging, with several function calls, etc.. 
When I added that to client, I added trace and debug, but it seems that debug is not used yet. I think I will put it back for possible later use.